src/main/java/org/jahia/modules/contentmanager/license/LicenseCheckFilter.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * ==========================================================================================
+ * =                            JAHIA'S ENTERPRISE DISTRIBUTION                             =
+ * ==========================================================================================
+ *
+ *                                  http://www.jahia.com
+ *
+ * JAHIA'S ENTERPRISE DISTRIBUTIONS LICENSING - IMPORTANT INFORMATION
+ * ==========================================================================================
+ *
+ *     Copyright (C) 2002-2019 Jahia Solutions Group. All rights reserved.
+ *
+ *     This file is part of a Jahia's Enterprise Distribution.
+ *
+ *     Jahia's Enterprise Distributions must be used in accordance with the terms
+ *     contained in the Jahia Solutions Group Terms &amp; Conditions as well as
+ *     the Jahia Sustainable Enterprise License (JSEL).
+ *
+ *     For questions regarding licensing, support, production usage...
+ *     please contact our team at sales@jahia.com or go to http://www.jahia.com/license.
+ *
+ * ==========================================================================================
+ */
+package org.jahia.modules.contentmanager.license;
+
+import org.jahia.services.render.RenderContext;
+import org.jahia.services.render.Resource;
+import org.jahia.services.render.filter.AbstractFilter;
+import org.jahia.services.render.filter.RenderChain;
+import org.jahia.services.render.filter.RenderFilter;
+import org.osgi.service.component.annotations.Component;
+
+/**
+ *
+ * Filter inserted in the render chain to validate the licence for CMM
+ * @author yousria
+ */
+@Component(service = RenderFilter.class, immediate = true, name = "LicenseCheckFilterCMM")
+public class LicenseCheckFilter extends AbstractFilter {
+
+    public LicenseCheckFilter() {
+        setPriority(1);
+        setApplyOnTemplates("content-manager");
+    }
```

Comment:
Rather use spring file to declare bean and set properties, as spring is already used in this module